src/gtk-3.20/scss/widgets/_view.scss

```diff
@@ -247,20 +247,20 @@
 
         header {
             button {
-                $_column_header_color: mix($fg_color, $base_color, .5);
+                $_column_header_color: $fg_color;
 
                 @extend %column_header_button;
 
                 color: $_column_header_color;
-                background-color: $base_color;
+                background-color: $bg_color;
                 font-weight: bold;
```

Comment:
thanks! when i see all the screenshots i think it will be also good to remove bold font style for gtk+3, what do you think?

Reply:
I didn't notice it's different from gtk+2, but yes, I agree.

Reply:
or mb we can use the button_bg and button_fg? i think that will be a good hint to state what they are the clickable area